apps/web/src/app/globals.css

```
@@ -942,6 +942,43 @@
 }
 
+/* §7.5b — Timeline fly-in motion (/club/geschiedenis). `TimelineReveal`
+   adds the `timeline-reveal` class to the wrapper on mount, then the
+   `--entered` modifier to each `[data-timeline-item]` as it crosses the
+   IntersectionObserver threshold. Cards slide in directionally (left side
+   from the left, right side from the right) and fade up; centred photos
+   (no `data-side`) default to a pure fade-up. The scoping under
+   `.timeline-reveal` means stories/no-JS render at the final state. */
+.timeline-reveal [data-timeline-item] {
+  --timeline-reveal-x: 0px;
+  opacity: 0;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)